sensu/resource_cluster_role_test.go

```diff
@@ -0,0 +1,46 @@
+package sensu
+
+import (
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/helper/resource"
+)
+
+func TestAccResourceClusterRole_basic(t *testing.T) {
+	resource.Test(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testAccResourceClusterRole_basic,
+				Check: resource.ComposeTestCheckFunc(
+					resource.TestCheckResourceAttr(
+						"sensu_cluster_role.cluster_role_1", "name", "cluster_role_1"),
+					resource.TestCheckResourceAttr(
+						"sensu_cluster_role.cluster_role_1", "rule.#", "2"),
+					resource.TestCheckResourceAttr(
+						"sensu_cluster_role.cluster_role_1", "rule.0.verbs.0", "*"),
```

Comment:
you check the `verbs` but should we also be checking the `resources`? (applies to both defined rules in the test)